src/main/java/org/sciborgs1155/robot/Ports.java

```diff
@@ -9,17 +9,9 @@ public static final class OI {
     public static final int RIGHT_STICK = 3;
   }
 
-  public static final class Drive {
-    public static final int FRONT_LEFT_DRIVE = 1;
-    public static final int REAR_LEFT_DRIVE = 38;
-    public static final int FRONT_RIGHT_DRIVE = 32;
-    public static final int REAR_RIGHT_DRIVE = 5;
-
-    public static final int FRONT_LEFT_TURNING = 31;
-    public static final int REAR_LEFT_TURNING = 36;
-    public static final int FRONT_RIGHT_TURNING = 3;
-    public static final int REAR_RIGHT_TURNING = 40;
-
-    public static final int PIGEON = 42;
+  public static final class Intake {
```

Comment:
so you have two subsystems, arm and intake, but you're using the intake ports for both of them. Also, this is more of an opinion thing but since it's acting as a shooter as well as an intake, how would you feel about calling intake flywheel instead?